chai-immutable.js

```diff
@@ -437,10 +438,15 @@ module.exports = function (chai, utils) {
    */
 
   assert.equal = function (actual, expected) {
+    /*
+     * It seems like we shouldn't actually need this check, however,
+     * `assert.equal` actually behaves differently than its BDD counterpart!
+     * Namely, the BDD version is strict while the "assert" one isn't.
+     */
```

Comment:
Yeah /:

Not too happy that chai does that but oh well. Incidentally, I wasn't sure what comment style to use here…this was the only thing that passed the linter!